app/controllers/helpers/BetaChecks.scala

```diff
@@ -42,4 +42,8 @@ trait BetaChecks {
 
   def changeDestinationBetaGuard(ern: String, arc: String, ver: Int)(implicit appConfig: AppConfig): Option[(String, Result)] =
     Some("tfeChangeDestination" -> Redirect(appConfig.emcsLegacyChangeDestinationUrl(ern, arc, ver)))
+
+  def preValidateBetaGuard(ern: String)(implicit appConfig: AppConfig): Option[(String, Result)] =
+    Some("tfePreValidate" -> Redirect(appConfig.emcsLegacyPreValidateUrl(ern)))
```

Comment:
Does this have to be an option? Saves doing `.get` in the controller. I'm not sure why all of these methods are optional?

Reply:
It doesn't have to be an Option, only did that to save overloading the functions in AuthActionHelper although I have now made a change to that at https://github.com/hmrc/emcs-tfe-frontend/pull/130/commits/b347a231fdcd4609b83c17c82624a65e5d8e05e7